readme.md

```diff
@@ -711,6 +711,7 @@
 - [Product Design](https://github.com/teoga/awesome-product-design#readme) - Design a product from the initial concept to production.
 - [Prisma](https://github.com/catalinmiron/awesome-prisma#readme) - Turn your database into a GraphQL API.
 - [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.
+  - [Education](https://github.com/mehdihadeli/awesome-software-architecture) - Articles and resources to learn and practice about software architecture, patterns, and principles.
```

Comment:
```suggestion
- [Education](https://github.com/mehdihadeli/awesome-software-architecture) - Articles and resources to learn and practice about software architecture, patterns, and principles.
```

The title seems confusing, given that there is a list of the same topic just above. I have no suggestion to change it, other than "Software Architecture", but that's be a clear duplicate and that's against the guidelines. How does your list compare to the previous one?